swagger/v1/api_spec.json

```diff
@@ -1602,8 +1602,8 @@
                   "teacher_reference_number": null,
                   "teacher_reference_number_validated": false,
                   "eligible_for_funding": null,
-                  "pupil_premium_uplift": true,
-                  "sparsity_uplift": false,
+                  "pupil_premium_uplift": null,
```

Comment:
These should be true and false. Active participants can't have null values in these fields anymore.